### ISSUE TITLE
CSS function with SCSS interpolations inside is no longer a correct CSS function

### ISSUE THREAD

**Issue by marcelweder** [stylelint-scss/stylelint-scss#975]
Since the [release of v6.2.0](https://github.com/stylelint-scss/stylelint-scss/releases/tag/v6.2.0) we get an error `function-no-unknown` with CSS functions like `calc`. In v6.2.0 this rule has been extended with a [detection if interpolations](https://github.com/stylelint-scss/stylelint-scss/pull/970) are present.

We do not understand why a CSS function with SCSS interpolations inside is no longer a correct CSS function. Unfortunately, we now have to extend the `scss/function-no-unknown` rule to exclude (ignore) the CSS functions.

Example for the reproduction:

```scss
$test-sass-variable: 5rem;

.test-rule {
  width: calc(100% - #{$test-sass-variable});
  max-width: calc(100% - 5rem);
}
```

| Before v6.2.0 |  New is an error |
| ------ | ------ |
| ![image](https://github.com/stylelint-scss/stylelint-scss/assets/1328308/f6587adb-417d-4e09-a5ce-44608a2446c8) | ![image](https://github.com/stylelint-scss/stylelint-scss/assets/1328308/1726961a-7873-4527-b86a-ad134d720898) |

Would be nice if you could correct this again. Thank you.

**Comment by kristerkari** [comment]
New version out that should fix the bug:
https://github.com/stylelint-scss/stylelint-scss/releases/tag/v6.2.1
